File: add/codesnippet/CSharp/t-system.componentmodel._99_1.cs
		try {
		    License licTest = null;
		    licTest = LicenseManager.Validate(typeof(Form1), this);
		}

		catch(LicenseException licE) {
		    Console.WriteLine(licE.Message);
		    Console.WriteLine(licE.LicensedType);
		    Console.WriteLine(licE.StackTrace);
		    Console.WriteLine(licE.Source);	
		}

File: add/codesnippet/CPP/t-system.componentmodel._99_1.cpp
      try
      {
         License^ licTest = nullptr;
         licTest = LicenseManager::Validate( Form1::typeid, this );
      }
      catch ( LicenseException^ licE ) 
      {
         Console::WriteLine( licE->Message );
         Console::WriteLine( licE->LicensedType );
         Console::WriteLine( licE->StackTrace );
         Console::WriteLine( licE->Source );
      }